Test build break when building release/uwp6.1

https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=1423221  

```
 Indexer.cs(68,29): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(103,39): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(141,39): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(178,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(212,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(245,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(281,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(323,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(360,34): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(397,42): error CS0619: 'ReadOnlySpan<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(431,42): error CS0619: 'ReadOnlySpan<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(464,42): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(500,42): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(655,36): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(692,36): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(739,39): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(782,39): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 Indexer.cs(829,39): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\Indexer.csproj]
 SpanBench.cs(133,37): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\SpanBench.csproj]
 SpanBench.cs(188,44): error CS0619: 'Span<byte>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\SpanBench.csproj]
 SpanBench.cs(246,39): error CS0619: 'Span<int>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\SpanBench.csproj]
 SpanBench.cs(308,40): error CS0619: 'Span<int>' is obsolete: 'Types with embedded references are not supported in this version of your compiler.' [E:\A\_work\543\s\tests\src\JIT\Performance\CodeQuality\Span\SpanBench.csproj]
```

--- a/tests/src/dirs.proj
+++ b/tests/src/dirs.proj
@@ -30,6 +30,10 @@
       <DisabledProjects Include="Performance\performance.csproj" />
       <DisabledProjects Include="Loader\classloader\generics\regressions\DD117522\Test.csproj" />
       <DisabledProjects Include="Loader\classloader\generics\GenericMethods\VSW491668.csproj" /> <!-- issue 5501 -->
+      <DisabledProjects Include="Interop\BestFitMapping\BestFitMapping.csproj" />
+      <DisabledProjects Include="JIT\Performance\CodeQuality\Span\Indexer.csproj" />
+      <DisabledProjects Include="JIT\Performance\CodeQuality\Span\SpanBench.csproj" />
+      <DisabledProjects Include="performance\linkbench\linkbench.csproj" />
       <DisabledProjects Include="JIT\Directed\nullabletypes\**\*.csproj" Condition="'$(RunningOnCore)' == 'true'" />
       <DisabledProjects Include="JIT\Performance\CodeQuality\Span\*.csproj" Condition="'$(RunningOnCore)' == 'true'" />
       <!-- The following project requires System.Reflection.Emit which is not supported on Linux, see issue corefx@4491 -->

